src/i18n/messages/fr-FR.ts

```diff
@@ -8,57 +8,57 @@ export const French: Messages = {
+  q2: "2. Comment puis-je jouer si je suis daltonien ou malvoyant?",
+  a2: "La section <button>Paramètres</button> contient un mode de jeu à contraste élevé.",
+  q3: "3. Qu'est-ce qui détermine la validité d'un pays dans le jeu?",
+  a3: "Globle s'appuie sur cette <a>convention</a> afin de déterminer ce qui constitue une tentative valide.",
+  q4: "4. Y a-t-il des pays autonomes mais non souverains dans le jeu?",
+  a4: "Certains territoires apparaitront avec une couleur neutre lorsque leur pays souverain est deviné, par exemple le Groenland pour le Danemark. L'emplacement de ces territoires n'affecte pas la couleur attribuée au pays souverain. La plupart des petits territoires n'apparaissent pas dans le jeu, par exemple Curaçao.",
+  q5: "5. J'ai trouvé le pays mystère d'aujourd'hui! Quand vais-je pouvoir rejouer?",
   a5: "Le pays mystère change et vos tentatives sont réinitialisées à minuit dans votre fuseau horaire.",
-  q6: "6. Est-ce que les noms alternatifs de pays sont acceptés ?",
+  q6: "6. Est-ce que les noms alternatifs de pays sont acceptés?",
   a6: "Il y a beaucoup de pays avec plusieurs noms acceptables. Certains noms alternatifs ou anciens sont acceptés, par exemple la Birmanie pour le Myanmar. Aussi, les acronymes sont acceptés pour les pays avec des noms composés de plusieurs mots, par exemple EAU pour les Émirats arabes unis.",
-  q7: "7. Un pays est manquant ou une frontière est incorrecte. Que puis-je faire ?",
-  a7: "La Géographie peut être un sujet sensible, et certaines frontières sont disputées. Si vous pensez qu'une correction doit être faite, s'il vous plait signalez une erreur sur {GitHub} ou envoyez-moi un DM sur {Twitter}.",
+  q7: "7. Un pays est manquant ou une frontière est incorrecte. Que puis-je faire?",
+  a7: "La géographie peut être un sujet sensible, et certaines frontières sont contestées. Si vous pensez qu'une correction s'impose, s'il vous plait veuillez m'en informer en ouvrant un billet (\"issue\") sur {GitHub} ou en m'envoyant un message privé sur {Twitter}.",
   GameTitle: "Jeu",
-  Game1: "Entrez le nom d'un pays ici",
+  Game1: "Entrez le nom d'un pays ici.",
   Game2: "Entrer",
   Game3:
     "Entrez le nom de n'importe quel pays pour faire votre première tentative.",
-  Game4: `Glissez, <span>["cliquez", "tap"]</span>, et zoomez sur le globe pour vous aider à trouver votre prochaine tentative.`,
+  Game4: `Glissez, <span>["cliquez", "tapez"]</span>, et agrandissez le globe pour vous aider à trouver votre prochaine tentative.`,
   Game5: "Tentative invalide",
   Game6: "Pays déjà tenté",
-  Game7: "Le pays mystère est {answer}!",
+  Game7: "Le pays mystère est: {answer}!",
   Game8: "Frontière la plus proche",
   StatsTitle: "Statistiques",
   Stats1: "Dernière victoire",
   Stats2: "Tentatives d'aujourd'hui",
   Stats3: "Parties gagnées",
   Stats4: "Série actuelle",
   Stats5: "Série max",
-  Stats6: "Moyenne de tentative",
-  Stats7: "Avg. Guesses",
```

Comment:
Here, the original translator left it in English probably because he want to share his stats of the day in English.  I haven't found anything similar in the other translations.